modules/javafx.controls/src/test/java/test/javafx/css/PropertySizeTest.java

```diff
@@ -248,10 +251,12 @@ public void ideal_relativeFontSizeNestedParentControlTest() {
         l3.setStyle("-fx-font-size: 0.25em");
         l4.setStyle("-fx-font-size: 0.25em");
 
-        // This should have been the behavior of font size calculation with
-        // nested set of parents and controls. We are not changing current behaviour to avoid
-        // regressing any applications that rely on current behaviour. Current behaviour can be
-        // observed in other -fx-font-size tests here.
+        // This should have been the behavior of font size calculation with nested
+        // set of parents and controls, that a Parent's font size is calculated with reference
+        // to its Parent and not grand parent. We are not changing current behaviour to avoid
```

Comment:
Very minor: grandparent.
Also very minor: you use both behavior and behaviour in this paragraph. Since this isn't public docs, I don't care which one you use, but you might want to pick one and stick with it.